src/MCPClient/lib/clientScripts/normalizeReport.py

```diff
@@ -0,0 +1,174 @@
+        </head>
+        <body>
+            <table cellpadding="10" width="100%">
+                <tr>
+                    <td></td>
+                    <td>
+                        <h1>Archivematica</h1>
+                        <hr />
+                        <h3>
+                            {{ name }} - {{ uuid }}<br />
+                            Normalization failure report - failures detected!
+                        </h3>
+                        <table cellpadding="10" border="1" width="100%">
+                            <thead>
+                                <tr>
+                                    <th>File name</th>
+                                    <th>File UUID</th>
+                                    <th>Exit code</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {% for type, tasks in failed_tasks.items %}
+                                    <tr>
+                                        <td colspan="3" style="background-color: #aaa; font-weight: bold;">
+                                            {{ type }} (total = {{ tasks.count }})
+                                        </td>
+                                    </tr>
+                                    {% for item in tasks %}
+                                        <tr>
+                                            <td>{{ item.filename }}</td>
+                                            <td>{{ item.fileuuid }}</td>
+                                            <td>{{ item.exitcode }}</td>
+                                        </tr>
+                                    {% endfor %}
+                                {% endfor %}
+                            </tbody>
+                        </table>
+                    </td>
+                    <td></td>
+                </tr>
+            </table>
+            <table class="footer-wrap">
+                <tr>
+                    <td></td>
+                    <td class="container">
+                        <div class="content">
+                            <table>
+                                <tr>
+                                    <td align="center">
+                                        <p>Archivematica</p>
+                                    </td>
+                                </tr>
+                            </table>
+                        </div>
+                    </td>
+                    <td></td>
+                </tr>
+            </table>
+        </body>
+    </html>
+"""
+
+
+def report(uuid):
+    """
+    Generate normalization report using Django's template module and send it to
+    every active user.
+    """
+    recipient_list = User.objects.filter(is_active=True).values_list('email', flat=True).exclude(email__in=['demo@example.com', ''])
+    if not recipient_list:
+        logger.error('Normalization report is not being sent because the recipient list is empty.')
```

Comment:
Is this an error case, or an expected do-nothing case? I lean towards the latter.